cscs-checks/mch/openacc_cuda_mpi_cppstd.py

```diff
@@ -93,10 +93,10 @@ def setflags(self):
                 ]
 
     @rfm.run_before('compile')
-    def cdt2006_pgi_workaround(self):
+    def cdt2008_pgi_workaround(self):
         cdt = os_ext.cray_cdt_version()
         if not cdt:
             return
 
-        if (self.current_environ.name == 'PrgEnv-pgi' and cdt == '20.06'):
+        if (self.current_environ.name == 'PrgEnv-pgi' and cdt == '20.08'):
```

Comment:
```suggestion
        if (self.current_environ.name == 'PrgEnv-pgi' and cdt >= '20.06'):
```

This will work because of the naming convention of the CDTs.